src/views/users/remove-from-group-button.tsx

```diff
@@ -24,6 +26,22 @@ export const RemoveFromGroupButton: FC<RemoveFromGroupButtonProps> = ({ group, u
         ({ isCritical }) => (isCritical ? { critical: true } : { success: true })
     )
 
+    const onClick = useCallback(
+        (_, event) => {
+            event.stopPropagation()
+            setShowConfirmationModal(true)
+        },
+        [setShowConfirmationModal]
+    )
+
+    const onCancel = useCallback(
+        (_, event) => {
+            event.stopPropagation()
+            setShowConfirmationModal(false)
+        },
+        [setShowConfirmationModal]
+    )
```

Comment:
These two do not need `useCallback` as the function is eventually just given to `onClick` props of the `Button` component. Contrary to the `useAddGroups` hook, this code is not meant for re-use.